pkg/operatorstatus/operatorstatus.go

```diff
@@ -10,123 +10,133 @@ import (
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/client-go/util/retry"
 
-	osapi "github.com/openshift/cluster-version-operator/pkg/apis/operatorstatus.openshift.io/v1"
+	configv1 "github.com/openshift/api/config/v1"
 	"github.com/operator-framework/operator-sdk/pkg/sdk"
 
 	"github.com/openshift/cluster-samples-operator/pkg/apis/samplesoperator/v1alpha1"
 )
 
-// CVOOperatorStatusHandler allows for wrappering sdk access to osapi.ClusterOperator
+// CVOOperatorStatusHandler allows for wrappering sdk access to configv1.ClusterOperator
 type CVOOperatorStatusHandler struct {
 	SDKwrapper OperatorStatusSDKWrapper
-	Namespace  string
 }
 
 // NewCVOOperatorStatusHandler is the default initializer function for CVOOperatorStatusHandler
-func NewCVOOperatorStatusHandler(namespace string) *CVOOperatorStatusHandler {
-	return &CVOOperatorStatusHandler{SDKwrapper: &defaultOperatorStatusSDKWrapper{}, Namespace: namespace}
+func NewCVOOperatorStatusHandler(clientGoWrapper *OperatorStatusSDKWrapper) *CVOOperatorStatusHandler {
+	handler := CVOOperatorStatusHandler{}
+	if clientGoWrapper != nil {
+		handler.SDKwrapper = *clientGoWrapper
+	} else {
+		handler.SDKwrapper = &defaultOperatorStatusSDKWrapper{}
+	}
+	return &handler
 }
 
 // OperatorStatusSDKWrapper is the interface that wrappers actuall access to github.com/operator-framework/operator-sdk/pkg/sdk
 type OperatorStatusSDKWrapper interface {
-	Get(name, namespace string) (*osapi.ClusterOperator, error)
-	Update(state *osapi.ClusterOperator) (err error)
-	Create(state *osapi.ClusterOperator) (err error)
+	Get(name string) (*configv1.ClusterOperator, error)
+	Update(state *configv1.ClusterOperator) (err error)
```

Comment:
UpdateStatus ?

Reply:
marked as resolved but i don't see a response or change?

Reply:
Sorry I'm updating this ... just forgot to post the comment